### ISSUE TITLE
Introduce real pagination than "load more"

### ISSUE THREAD

**Issue by mmmavis** [mozilla/network-pulse#631]
We now have a good amount of entries and navigating to a particular entry in list view starts to become troublesome. Users might have to click on "load more" a bunch of times to spot the entry they are looking for. We should probably move away from "load more" to real pagination. This will make sharing list of entries easier too (unique url for each page).

/cc @xmatthewx @hannahkane @Pomax

**Comment by Pomax** [comment]
I just ran into this too, with 500 test entries that were all set to "pending"... the "view more" button is not a super useful way to deal with that many entries =)

"prev/next" buttons with pagination blocks (`[1] [2] [3] ... [16] [17] [18]`) and a clear visual indication of which page we're on would be really nice to have.

**Comment by xmatthewx** [comment]
Let's discuss!

**Comment by alanmoo** [comment]
I'm in favor of pagination across the board.

**Comment by kristinashu** [comment]
I prefer the "load more" button. Pagination generally seems old school and not used on new social media type sites with user-generated content.

[Here is some user testing done around this topic](https://www.smashingmagazine.com/2016/03/pagination-infinite-scrolling-load-more-buttons/) which supports the load more button. Note this is for ecommerce but I think a lot of the points carry over (eg test subjects generally perceived pagination to be slow, easier to tap on mobile, less cognitive load for users...).

tl;dr "Subjects generally browsed more products on websites with a “Load more” button than on those with pagination links, but because loading additional products still required an active choice and click, subjects did tend to read the displayed items much more closely than on websites with infinite scrolling."

**Comment by Pomax** [comment]
Given the preferences expressed, should we close this issue?

**Comment by alanmoo** [comment]
Fine by me, I'm only at a 💩x 1 given for pagination.

**Comment by Pomax** [comment]
hahaha, that poor emoji

**Comment by mmmavis** [comment]
Heh forgot to comment. Was gonna +1 on @alanmoo 's comment 😂
